mysite/ct/tests.py

```diff
@@ -691,3 +691,30 @@ def test_continue_partial(self, url_name, url_post_data):
         self.assertFalse(PartialHashTable.objects.filter(token=token).exists())
         result = self.client.get(continue_url)
         self.assertEqual(result.status_code, 404)
+
+
+    def test_get_on_anonymous_user(self, *args):
```

Comment:
Emmmm..... I was playing with different tests and decorators and forgot to delete it